src/index.css

```diff
@@ -16,12 +16,15 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+.result {
+  background-color: #848693;
+  text-align: end;
+}
+
+.operator {
+  background-color: #f39148 !important;
+}
+
+.numbers {
+  background-color: #e0e0e0;
```

Comment:
- [ ] Please correct the dimensions of the Calculator to look similar to the given design, this is a desktop web page 👇🏽 

| Original design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/219791470-8f88d747-3c24-4109-a11b-d645874babce.png) | ![image](https://user-images.githubusercontent.com/58642949/219791584-52ab6b83-b528-42e9-959d-16b42be80aee.png) |